packages/flutter/lib/src/animation/animation_style.dart

```diff
@@ -28,10 +28,10 @@ import 'tween.dart';
 @immutable
 class AnimationStyle with Diagnosticable {
   /// Creates an instance of Animation Style class.
-  AnimationStyle({this.curve, this.duration, this.reverseCurve, this.reverseDuration});
+  const AnimationStyle({this.curve, this.duration, this.reverseCurve, this.reverseDuration});
```

Comment:
I am surprised that the `prefer_const_constructors_in_immutables` lint didn't complain about the missing non-const constructor. Could you file an issue at https://github.com/dart-lang/sdk/issues for this (if you can, they'd appreciate a small code snippet that reproduces the lint not triggering)?